docs/ref/forms/fields.txt

```diff
@@ -1077,14 +1079,24 @@ objects (in the case of ``ModelMultipleChoiceField``) into the
 
 For more complex uses, you can specify ``queryset=None`` when declaring the
 form field and then populate the ``queryset`` in the form's ``__init__()``
-method::
+method:
+
+.. code-block:: python
```

Comment:
The default highlighting should work without this unless the code block has a syntax error that causes the parsing to fail.